ha/core/event_analyzer/parser/parser.py

```diff
@@ -168,8 +170,26 @@ def __init__(self):
         self.cluster_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}cluster_id")
         self.site_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}site_id")
         self.rack_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}rack_id")
+
+        _index = 'cortx'
+        ConfigManager._safe_load(_index, "yaml:///etc/cortx/cluster.conf")
+        ConfigManager.init(None)
+
+        self.data_node_ids = ConftStoreSearch.get_data_pods(_index)
+        self.server_node_ids = ConftStoreSearch.get_server_pods(_index)
+        self.control_node_ids = ConftStoreSearch.get_control_nodes(_index)
         Log.info("ClusterResource Parser is initialized ...")
 
+    def get_functional_type(self, node_id):
```

Comment:
Please add docstring and sample on returns.

Reply:
fixed